src/modules/RA/DataLayer/components/tabs/PopupTab/PopupFielSelect.js

```diff
@@ -10,16 +10,17 @@ const FieldSelect = ({
   path,
   fields,
   selectable,
-  selected,
+  selected = '',
```

Comment:
`selected` should be null or number ?

Reply:
I avoided null. In this component, it will be use as a value for the `select` and I want the component to stay "controlled".